### ISSUE TITLE
Possible bug: Panorama template stack not setting default vsys correctly

### ISSUE THREAD

**Issue by erikpaasonen** [terraform-providers/terraform-provider-panos#85]
Hello,
When we use a `panos_panorama_template_stack` resource, Terraform applies successfully. However, when we login to the GUI to perform a Commit, it fails to commit with the following error message:
```
Removing the default VSYS will result in VSYS configuration not being pushed to firewalls with single virtual system.
```
The assumption based on the documentation (specifically, the absence of a "vsys" argument) is that we can ignore the VSYS entirely. It seems that the template stack is either staging the deletion of the VSYS or setting it to a different value than `vsys1`.

Ref: https://www.terraform.io/docs/providers/panos/r/panorama_template_stack.html

Separate side note -- this documentation page is missing an Attribute Reference section header... :)

**Comment by shinmog** [comment]
@erikpaasonen 

Ok, I think this is a bug on my part - the API does weird stuff with the template stack creation in that it creates the template, then does a follow-up operation to set the default vsys, but if you try to combine these two things into one operation, the API rejects it.  So I removing setting the default vsys since the API wasn't tolerating it.  I didn't try doing it exactly like how the GUI was doing it - splitting it into two operations.  I'll have to play with this a bit...  If this is the case, then I'll fix it in the next release.

**Comment by erikpaasonen** [comment]
@shinmog  Thank you! Our team is anxiously awaiting the fix!

**Comment by shinmog** [comment]
@erikpaasonen 

After some trial and error, Panorama template stacks' `default_vsys` can only be non-empty if there is at least one template in the template stack.  I'll be updating the documentation when I fix this as well.

**Comment by erikpaasonen** [comment]
@shinmog Thank you for addressing this! Is there a way we can test this yet, or do we need to wait for the next release of the TF provider? Pardon my ignorance of the provider release process.

**Comment by shinmog** [comment]
@erikpaasonen 
No worries.  You can build the provider yourself by just following the instructions on the main page of this repo, but that isn't strictly necessary now as I'm hoping to do a mini release today, just finishing up the acceptance tests for another small addition I'm adding.

**Comment by shinmog** [comment]
@erikpaasonen 
This is fixed in the new 1.4.1 provider.  A template stack's `default_vsys` can now be configured, and this should fix the PAN-OS complaint you were getting before.

In practice, a template's `default_vsys` is now `Computed: true`, so it will either be an empty string or "vsys1".  When you configure the `default_vsys` on a template stack, however, it must be set to a vsys that is in one of the templates that are attached to the template stack.  From a Terraform perspective, this will most likely mean that you're either setting a template stack's `default_vsys` to `vsys1` or leaving it blank.
